app/templates/main/upload.html

```diff
@@ -15,110 +15,114 @@
-                "rows": [
-                  [
-                    {
-                      "text": "Fund"
-                    },
-                    {
-                      "html": fund
-                    }
-                  ],
-                  [
-                    {
-                      "text": "Local authority"
-                    },
-                    {
-                      "text": localAuthorities
-                    }
-                  ],
-                  [
-                    {
-                      "text": "Reporting period"
-                    },
-                    {
-                      "text": reporting_period
-                    }
-                  ]
-                ]
-            }) }}
-
-        {% else %}
-            <h2 class="govuk-heading-m">There are errors in your return</h2>
-            <p class="govuk-body">The following errors have been found in your return.</p>
-            {{ errorTable(validation_errors) }}
-            <h2 class="govuk-heading-m">Errors fixed?</h2>
-            <h3 class="govuk-heading-s">Re-upload your data return</h3>
-        {% endif %}
-
-        <div class="upload-data-container govuk-!-margin-top-5">
-            <form method="post" action="{{ url_for('main.upload') }}" enctype="multipart/form-data">
-                {% if not pre_error %}
-                    <h2 class="govuk-heading-m">Upload your data return</h2>
-
-                    {{ govukFileUpload({'id': 'ingest_spreadsheet',
-                     "hint": {"text": ""},
-                     "label": {"text": ""},
-                     'name': 'ingest_spreadsheet'}) }}
                 {% else %}
-                     <h2 class="govuk-heading-m">Upload your data return</h2>
-
-                    {{ govukFileUpload({'id': 'ingest_spreadsheet',
-                     "hint": {"text": "Upload an Excel file"},
-                     "label": {"text": ""},
-                     "errorMessage": {"text": pre_error[0]},
-                     'name': 'ingest_spreadsheet'}) }}
+                    <h2 class="govuk-heading-m">There are errors in your return</h2>
+                    <p class="govuk-body">The following errors have been found in your return.</p>
+                    {{ errorTable(validation_errors) }}
+                    <h2 class="govuk-heading-l">Errors fixed?</h2>
+                    <h3 class="govuk-heading-m">Re-upload your data return</h3>
                 {% endif %}
-                <p class="govuk-body">When you upload your return, we’ll check it for missing data and formatting errors.</p>
 
-                {{ govukButton({"element": "button",
-                "text": "Upload and check for errors",
-                "attributes": {
-                    "aria-controls": "example-id",
-                    "data-tracking-dimension": 123} }) }}
-            </form>
-            {{ helpLinksDropdown() }}
-        </div>
+                <div class="upload-data-container govuk-!-margin-top-5">
```

Comment:
Same her I think

Reply:
this is correct - if you check the closing tags you can see it lines up